single-line-diagram/single-line-diagram-core/src/main/java/com/powsybl/sld/svg/DiagramStyles.java

```diff
@@ -46,6 +46,14 @@ public final class DiagramStyles {
     public static final String ANGLE = STYLE_PREFIX + "angle";
     public static final String FEEDER_INFO = STYLE_PREFIX + "feeder-info";
 
+    public static final String ARROW_ANIMATION = STYLE_PREFIX + "arrow-animation";
+
+    public static final String ARROW_ANIMATION_SPEED_1 = STYLE_PREFIX + "arrow-animation-speed-1";
```

Comment:
We have to decide how to discretize the power values and maybe include  "no value" or "zero power" cases.